github-event/handler.go

```diff
@@ -320,7 +319,7 @@ func getCustomers(customerURL string) ([]string, error) {
 
 func readBool(key string) bool {
```

Comment:
This should take an override like `readBool(key string, defaultVal bool)` otherwise it's not the equivalent of `readBool` elsewhere in OpenFaaS where null/empty meant false. Alternatively change `readBool` to something else like a more specific validateHmac() or hmacEnabled().

Reply:
There is `HmacEnabled()` in SDK, should I create `HmacDisabled()` or change the logic of `HmacEnabled()`? Which way should I go with ?

Reply:
Done ✔️